spring-cloud-gcp-samples/spring-cloud-spanner-r2dbc-samples/pom.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <!-- Your own application should inherit from spring-boot-starter-parent -->
+    <artifactId>spring-cloud-gcp-samples</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.5.4-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>spring-cloud-spanner-r2dbc-samples</artifactId>
+  <name>Spring Framework on Google Cloud Code Sample - Spanner-r2dbc</name>
+
+  <properties>
+    <sonar.skip>true</sonar.skip>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>cloud-spanner-r2dbc</artifactId>
+        <version>${cloud-spanner-r2dbc.version}</version>
+      </dependency>
+
+      <!-- Until Spring Boot picks up Spring Data R2DBC 1.4.1, drivers
```

Comment:
This comment seems out of date. Is the workaround still needed now that Spring Data R2DBC 1.5.13 is being used in the latest Spring Data BOM, or should this comment be reworked to clarify the remaining constraint?

https://github.com/spring-projects/spring-data-bom/releases/tag/2021.2.13

Reply:
@zhumin8 - This looks outdated, shall I remove it?

Reply:
Yes, please do.

Reply:
done